balboa-http/src/main/scala/com/socrata/balboa/server/Main.scala

```diff
@@ -1,48 +1,14 @@
 package com.socrata.balboa.server
 
-import javax.servlet.http.HttpServletRequest
+import org.eclipse.jetty.server.Server
+import org.slf4j.LoggerFactory
+import org.eclipse.jetty.webapp.WebAppContext
+import org.scalatra.servlet.ScalatraListener
 
-import com.socrata.balboa.metrics.config.Configuration
-import com.socrata.balboa.server.rest.{EntitiesRest, MetricsRest, VersionRest}
-import com.socrata.http.routing.{HttpMethods, SimpleRoute, SimpleRouter}
-import com.socrata.http.server.implicits._
-import com.socrata.http.server.responses._
-import com.socrata.http.server.{Filter, HttpResponse, Service, SimpleFilter, SocrataServerJetty}
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.log4j.PropertyConfigurator
+object Main extends App {
+  val logger = LoggerFactory.getLogger(getClass)
```

Comment:
You should be able to use:

    with StrictLogging

instead of an explicit logger in the class.